### PR TITLE
Tag: Price Tag and Rating Tag components

### PR DESCRIPTION
**Pull request by carizzuti**
Different tags for the price and the rating.

image

### CODE REVIEW

File: mobile/src/components/tag/Tag.style.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    flexDirection: 'row',
+    margin: 20,
+  },
+  priceText: {
+    fontSize: size.rg,
+    color: color.black,
+  },
+  dollarSign: {
+    color: color.primary,
+    fontSize: size.rg,
```
Comment (Nap1400) [inline mobile/src/components/tag/Tag.style.tsx:23]: This will be a different size when implemented in the screens, correct

**Review by Nap1400 — APPROVED** [review]

File: mobile/src/components/tag/Tag.style.tsx (hunk 1 above)
Comment (carizzuti) [reply mobile/src/components/tag/Tag.style.tsx:22]: Actually it looks like I forgot to add a style prop

**Comment by Dulaya** [comment]
* Don't forget to export them from index.ts
* In container, replace paddingHorizontal: size.xs with padding: size.sm (8). Both horizontal and vertical have about same padding. 
* In container, remove borderWidth. 
* Create another type and prop named shadowEnable?: boolean. If shadowEnable, then show shadow. Else, hide shadow. Inside Tag.tsx:

        <View
          style={[
            styles.container,
            containerStyle,
            shadowEnable && {shadowColor: color.grayMid, elevation: 5},
          ]}>
          {renderTag()}
        </View>

* Optional: add margin: size.xxs to the numbers. Play around to find best look.

image

**Review by Dulaya — APPROVED** [review]

File: mobile/src/theme/color/color.ts [hunk 2]
```diff
@@ -12,5 +12,5 @@ export const color: Color = {
   textRegularBG: '#E9E9E940',
   yellow: '#FFC529',
   red: '#FE724C',
-
+  transparentBlack: 'rgba(0, 0, 0, 0.1)',
```
Comment (mshago) [inline mobile/src/theme/color/color.ts:15]: do you need this color?

**Review by lasharela — CHANGES_REQUESTED** [review]

File: mobile/src/theme/color/color.ts (hunk 2 above)
Comment (carizzuti) [reply mobile/src/theme/color/color.ts:15]: At first I was using it but I changed the shadow style

**Review by mshago — APPROVED** [review]

File: mobile/src/components/tag/Tag.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React, {FC} from 'react';
+import {Text, View} from 'react-native';
+import {size} from '../../theme';
+import {Icon} from '../icon/Icon';
+import {styles} from './Tag.style';
+import {TTagProps} from './Tag.type';
+
+export const Tag: FC<TTagProps> = ({
+  rating = 4.5,
```
[the remark this answers is not in the document]
Comment (carizzuti) [reply mobile/src/components/tag/Tag.tsx]: I removed that, made it optional.

**Review by aahenry51 — APPROVED** [review]

**Pull request merged**